python/vyos/ifconfig/tunnel.py

```diff
@@ -244,3 +247,218 @@ def _create(self):
         if '6rd-relay-prefix' in self.config:
             set6rd += ' 6rd-relay-prefix {6rd-relay-prefix}'
         self._cmd(set6rd.format(**self.config))
+
+class ERSpanIf(_Tunnel):
+    """
+    Sit: Simple Internet Transition
+
+    For more information please refer to:
+    https://git.kernel.org/pub/scm/network/iproute2/iproute2.git/tree/ip/link_iptnl.c
+    """
+
+    default = {'type': 'erspan'}
+    options = ['local', 'remote', 'dev', 'key', 'erspan_ver']
+    
+    def _create(self):
+        # do not call _Tunnel.create, building fully here
+        
+        conf = Config()
+        base = ['interfaces', 'tunnel']
+        tunnel = get_interface_dict(conf, base)
```

Comment:
We do not query the configuration backend from a backend script. This must be done in the interfaces-tunnel.py frontend. Layers should not be intermixed.